en/installation.rst

```diff
@@ -528,6 +528,32 @@ these steps:
 Once the web.config file is created with the correct IIS-friendly rewrite rules,
 CakePHP's links, CSS, JavaScript, and rerouting should work correctly.
 
+Lighttpd
+--------
+Lighttpd does not make use of .htaccess files like Apache, so it is necessary to add a url.rewrite-once configuration in **conf/lighttpd.conf**. This is a complete example of adding a cake php application to lighttpd configuration:
+
+.. code-block:: php
+    
+    server.modules += (
+        "mod_alias",
+        "mod_cgi",
+        "mod_rewrite"
+    )
+    
+    # Directory Alias
+    alias.url       = ( "/TestCake" => "C:/Users/Nicola/Documents/TestCake" )
+    
+    # CGI Php
+    cgi.assign      = ( ".php" => "c:/php/php-cgi.exe" )
+    
+    # Rewrite Cake Php (with /TestCake subdomain)
+    url.rewrite-once = (
+        "^/TestCake/(css|files|img|js|stats)/(.*)$" => "/TestCake/webroot/$1/$2",
+        "^/TestCake/(.*)$" => "/TestCake/webroot/index.php/$1"
+    )
+
+The above lines include also the php configuration and an alias configuration for an example application to publish on the subdomain /TestCake.
```

Comment:
Is this a subdomain? It looks like a path.

Reply:
Ok, a path.